src/components/Banner.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled';
+
+export default function Banner() {
+  return (
+    <Wrapper>
+      <BannerBox>
+        <TopText>카카오테크 캠퍼스 3기 여러분</TopText>
+        <BottomText>프론트엔드 2단계 과제 화이팅! 🎉</BottomText>
+      </BannerBox>
+    </Wrapper>
+  );
+}
+
+const Wrapper = styled.section`
+  width: 100%;
+  background-color: ${({ theme }) => theme.colors.background.default};
+  padding: 16px 0;
+  margin: 0 auto;
+  padding: 0 16px;
+  box-sizing: border-box;
+`;
+
+
+const BannerBox = styled.div`
+  background-color: ${({ theme }) => theme.colors.kakao.yellow.default};
+  border-radius: 16px;
+  padding: 16px;
+`;
```

Comment:
```suggestion
const spacing = "16px";

const Wrapper = styled.section`
  width: 100%;
  background-color: ${({ theme }) => theme.colors.background.default};
  padding: ${spacing} 0;
  margin: 0 auto;
  padding: 0 ${spacing};
  box-sizing: border-box;
`;


const BannerBox = styled.div`
  background-color: ${({ theme }) => theme.colors.kakao.yellow.default};
  border-radius: ${spacing};
  padding: ${spacing};
`;
```

javascript 변수를 이용해서 이렇게 표현할 수도 있겠네요!